packages/flutter/test/material/app_test.dart

```diff
@@ -6,7 +6,7 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:flutter/material.dart';
```

Comment:
Does dartfmt not normally work for this stuff? It's giving me errors on the PopupMenuButton PR also.